tests/alive-tv/attrs/dereferenceable-shrink.srctgt.ll

```diff
@@ -6,6 +6,6 @@ define void @src(i8* %p) {
 }
 
 define void @tgt(i8* %p) {
-  call void @f(i8* dereferenceable(4) %p)
```

Comment:
This test wasn't shrinking the dereferenceable size (which isn't what the file name is saying).
alive2test.py is already running identity tests for .srctgt tests, so I fixed this test to shrink this.